taskcluster/ci/merge-mono/kind.yml

```diff
@@ -0,0 +1,119 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - translations_taskgraph.transforms.from_datasets:mono
+    - translations_taskgraph.transforms.command_context_from_params:transforms
+    - taskgraph.transforms.job:transforms
+    - translations_taskgraph.transforms.cache:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - clean-mono
+    - toolchain
+
+task-defaults:
+    label: merge-mono-{locale}
+    description: merge mono for {locale}
+    attributes:
+        stage: merge-mono
+        cache:
+            type: merge-mono
+            resources:
+                - pipeline/clean/merge-mono.sh
+
+    dataset-config:
+        substitution-fields:
+            - label
+            - description
+            - name
+            - treeherder.symbol
+            - worker.env
+            - fetches
+            - dependencies
+
+    worker-type: b-linux-large
+
+    worker:
+        docker-image: { "in-tree": "train" }
+        max-run-time: 3600
+        artifacts:
+            - name: public/build
+              path: /builds/worker/artifacts
+              type: directory
+        env:
+            LOCALE: "{locale}"
+            COMPRESSION_CMD: zstdmt
+
+    # Don't run unless explicitly scheduled
+    run-on-tasks-for: []
+
+    treeherder:
+        symbol: "{locale}"
+        platform: merge-mono/opt
+
+    run:
+        using: run-task
+        command:
+            - bash
+            - -c
+            # Arguments are:
+            # 1) output directory
+            # 2) max_sent (sent to shuf -n to output at most max_sent lines)
+            # 3) input files
+            - >-
+                export BIN=$MOZ_FETCHES_DIR &&
+                $VCS_PATH/pipeline/clean/merge-mono.sh
+                /builds/worker/artifacts/mono.$LOCALE.zst
+                {max_sent}
+                $MOZ_FETCHES_DIR/*.zst
+
+    fetches:
+        toolchain:
+            - preprocess
+        clean-mono:
+            - artifact: "{dataset_sanitized}.{locale}.zst"
```

Comment:
Could this be more than one artifact? Like if we have datasets for different years?